### ISSUE TITLE
NTR: Phosphomannomutase enzyme activity low in leukocytes, fibroblasts, or liver (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8135]
New term request Phosphomannomutase enzyme activity low in leukocytes, fibroblasts, or liver (segal_040409153952): PMM  UMLS: 1859346

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000781
- label: Reduced tissue phosphomannomutase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:11343337
- def: Activity of the enzyme phosphomannomutase (EC 5.4.2.8) in tissues below the lower limit of normal. The activity can be measured in multiple tissues including leukocytes, fibroblasts, and liver.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8135
- synonyms:
